task8.js

```diff
@@ -0,0 +1,11 @@
+function f(num1, num2) {
+  if (num2) {
+    return num1 + num2;
+  }
+  return function (num2) {
+    return num1 + num2;
+  };
+}
+
+console.log(f(2, 3));
```

Comment:
Какой результат будет, если я вторым аргументом передам `0`?
```suggestion
console.log(f(2, 0));
```